TrackletAlgorithm/Constants.h

```diff
@@ -200,6 +205,80 @@ constexpr unsigned int nbits_vmte_extra[trklet::N_LAYER + trklet::N_DISK] = {0,
 constexpr unsigned int nbits_maxvm = 5; // number of bits needed for max number of VMs per layer/disk (max number is 32)
 constexpr unsigned int nbits_maxvm_overlap = 4; // Overlap
 
+namespace TC {
+////////////////////////////////////////////////////////////////////////////////
+// Typedefs, enums, and constants needed by TrackletCalculator.
```

Comment:
@aryd Since we're adding this namespace to Constants.h, I would suggest renaming the namespace, just to remove historical references.
```suggestion
namespace TP {
////////////////////////////////////////////////////////////////////////////////
// Typedefs, enums, and constants needed by TrackletProcessor.
```

Reply:
I did change this - it had a large number of knock-on effects.